eos-tech-support/eos-write-live-image

```diff
@@ -153,6 +153,7 @@ if [ "$ISO" ]; then
         [mcopy]='mtools'
         [mkisofs]='genisoimage'
         [grub-mkimage]='grub-common'
+        [mksquashfs]='squashfs-tools'
```

Comment:
In theory we should add xorriso, dosfstools, mtools, genisoimage, squashfs-tools to the package dependencies, or else this path of the script will not work on unconverted Endless OS. But I'd expect ISO generation to move into the image builder in due course, so this would be an unnecessary 3MB in the image.